core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -4012,6 +4012,14 @@ object ZIO extends ZIOCompanionPlatformSpecific {
       else ZIO.fail(::(es.head, es.tail.toList))
     }
 
+  /**
+   * @see See [[zio.ZIO.validatePar]]
+   */
+  def validatePar_[R, E, A, Collection[+Element] <: Iterable[Element]](in: Collection[A])(
+    f: A => ZIO[R, E, Any]
+  )(implicit bf: BuildFrom[Collection[A], Any, Collection[Any]], ev: CanFail[E]): ZIO[R, ::[E], Unit] =
+    validatePar(in)(f).unit
```

Comment:
In the success case here there is no need to build the collection if we are going to immediately discard it. I think this will also let us to remove the `BuildFrom` constraint and accept any `Iterable` instead of `Collection[+Element] <: Iterable[Element]`, which will allow this operator to work with some other collection types like arrays and sets.

Reply:
good point.

Reply:
Great improvement on the signature! We can still avoid unnecessarily creating the collection by implementing in terms of `partitionPar` directly and returning the unit value if there are no errors.

Reply:
Right, addressed in latest commit.